webgpu.h

```diff
@@ -2086,6 +2598,20 @@ typedef struct WGPUDepthStencilState {
     float depthBiasClamp;
 } WGPUDepthStencilState WGPU_STRUCTURE_ATTRIBUTE;
 
+#define WGPU_DEPTH_STENCIL_STATE_INIT _wgpu_MAKE_INIT_STRUCT(WGPUDepthStencilState, { \
+    /*.nextInChain=*/NULL _wgpu_COMMA \
+    /*.format=*/WGPUTextureFormat_Undefined _wgpu_COMMA \
+    /*.depthWriteEnabled=*/{} _wgpu_COMMA \
```

Comment:
This needs to default to `WGPUOptionalBool_Undefined` because that isn't the zero-value. (We do this weird thing so that C++ and stdbool.h `bool`s will cast correctly to `WGPUOptionalBool`.)

Reply:
Ok! I added this directly in the generator, rather than manually specifying a default value to "undefined" for all use cases of optional bool, so that this rule is strongly enforced